docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>I think it would be better if we directly print the dataset element:</p><pre class="ql-syntax" spellcheck="false">for item in dataset.take(1):
    print(item)

</pre><p>This reduces the lengthy syntax and the reader's can gain more information on the tensor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>